ext/pg.h

```diff
@@ -315,6 +315,7 @@ VALUE pg_obj_to_i                                      _(( VALUE ));
 VALUE pg_tmbc_allocate                                 _(( void ));
 void pg_coder_init_encoder                             _(( VALUE ));
 void pg_coder_init_decoder                             _(( VALUE ));
+void pg_coder_mark 																		 _(( t_pg_coder * ));
```

Comment:
Parameters are moved somehow to the next line.
Since you suggested removing this function and added this comment, so, finally, better is to remove this function, correct?